pom.xml

```diff
@@ -103,6 +108,12 @@
                 <groupId>com.microfocus.darwin</groupId>
                 <artifactId>settings-service-client</artifactId>
                 <version>1.5.0-610</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>com.github.workerframework</groupId>
+                        <artifactId>worker-api</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
Why are we excluding `worker-api`?

Reply:
that was wrong, the dependency came from caf-dependency-management-bom which has now been upgraded. I left worker-api listed as it's a transient dependency